### ISSUE TITLE
Add a new overloaded method to ActorRef with default sender as noSender.

### ISSUE THREAD

**Issue by hepin1989** [akka/akka#25778]
I found this's a bit annoying when I have to write code like this in Java 
```
    managerRef.tell(StartConsumerGroup.getInstance(),
                    ActorRef.noSender());
```
If there is  a method with only take one parameter and leave the sender as `ActorRef.noSender()`,that will make my life a little easier .

then I could just write 
```
    managerRef.tell(StartConsumerGroup.getInstance());
```

This is especially handy when you are interacting with the outside world of an ActorSystem.

**Comment by johanandren** [comment]
I don't think this is a good idea because it would make it extremely easy to forget passing the actor self as sender when you send messages from an actor.

**Comment by hepin1989** [comment]
But when you are integrating with the outside world, it's always cumbersome too:( and people will still whether or not passing wrong sender right?

**Comment by raboof** [comment]
Thanks for posting these 'code ergonomy' tickets.

In this case I agree with @johanandren though, having an `ActorRef#tell(target)` that defaults to `noSender` makes it really easy to forget populating the `sender`.

I could imagine having a `void tell(targetActorRef, message)` on `AbstractActor`, that would be less error-prone. Having 'multiple ways to do `tell`' would be confusing though, so I'm not sure it is worth it.

With the current API, you can make this a little less cumbersome by doing an `import static akka.actor.ActorRef.noSender;`: then it's just `managerRef.tell(msg, noSender())` which might be less painful.

**Comment by hepin1989** [comment]
I am currently using Akka with Spring (I have to do it at work, using Java), and found something that not very convenient.
Or we could introduce a new method name, kind of `gossip` or `tellAnonymously` like thing?

**Comment by hepin1989** [comment]
@patriknw should mark this as `won't fix`

**Comment by patriknw** [comment]
ok, that is kind of implicit when not assigning the milestone on a closed ticket
